Detectors/CPV/calib/include/CPVCalib/CalibParams.h

```diff
@@ -0,0 +1,69 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \class CalibParams
+/// \brief CCDB container for the full set of CPV calibration coefficients
+/// \author Dmitri Peresunko, RRC Kurchatov institute
+/// \since Aug. 1, 2019
+///
+///
+
+#ifndef CPV_CALIBPARAMS_H
+#define CPV_CALIBPARAMS_H
+
+#include <array>
+#include "TObject.h"
+
+class TH2;
+
+namespace o2
+{
+
+namespace cpv
+{
+
+class CalibParams
+{
+ public:
+  /// \brief Constructor
+  CalibParams() = default;
+
+  /// \brief Constructor for tests
+  CalibParams(int test);
+
+  /// \brief Destructor
+  ~CalibParams() = default;
+
+  /// \brief Get High Gain energy calibration coefficients
+  /// \param cellID Absolute ID of cell
+  /// \return high gain energy calibration coefficient of the cell
+  float getGain(short cellID) const { return mGainCalib.at(cellID); }
```

Comment:
``at`` getter checks for the index range, for the correct code it is not needed but just takes time.